test/clj_test_containers/core_test.clj

```diff
@@ -13,6 +13,20 @@
       (is (some? (:mapped-ports initialized-container)))
       (is (some? (get (:mapped-ports initialized-container) 5432)))
       (is (nil? (:id stopped-container)))
+      (is (nil? (:mapped-ports stopped-container)))))
+
+  (testing "Testing basic testcontainer image creation from docker file"
+    (let [container (create-from-docker-file {:env-vars
```

Comment:
The env vars are never used or checked, are they? In this case I would remove them from the test setup, to not confuse any readers